Content.Client/RoundEnd/RoundEndSummaryWindow.cs

```diff
@@ -1,87 +1,104 @@
+            roundEndTabs.AddChild(MakePlayerManifestoTab(info));
+
+            Contents.AddChild(roundEndTabs);
+
+            OpenCentered();
+            MoveToFront();
+        }
+
+        private BoxContainer MakeRoundEndSummaryTab(string gamemode, string roundEnd, TimeSpan roundDuration)
+        {
+            var roundEndSummaryTab = new BoxContainer
             {
                 Orientation = LayoutOrientation.Vertical,
                 Name = Loc.GetString("round-end-summary-window-round-end-summary-tab-title")
             };
 
-            //Tab for listing  unique info per player.
-            PlayerManifestoTab = new BoxContainer
+            var roundEndSummaryContainerScrollbox = new ScrollContainer
             {
-                Orientation = LayoutOrientation.Vertical,
-                Name = Loc.GetString("round-end-summary-window-player-manifesto-tab-title")
+                VerticalExpand = true
+            };
+            var roundEndSummaryContainer = new BoxContainer
+            {
+                Orientation = LayoutOrientation.Vertical
             };
-
-            RoundEndWindowTabs = new TabContainer();
-            RoundEndWindowTabs.AddChild(RoundEndSummaryTab);
-            RoundEndWindowTabs.AddChild(PlayerManifestoTab);
-
-            Contents.AddChild(RoundEndWindowTabs);
-
             //Gamemode Name
             var gamemodeLabel = new RichTextLabel();
-            gamemodeLabel.SetMarkup(Loc.GetString("round-end-summary-window-gamemode-name-label", ("gamemode",gm)));
-            RoundEndSummaryTab.AddChild(gamemodeLabel);
+            gamemodeLabel.SetMarkup(Loc.GetString("round-end-summary-window-gamemode-name-label", ("gamemode", gamemode)));
+            roundEndSummaryContainer.AddChild(gamemodeLabel);
 
             //Round end text
             if (!string.IsNullOrEmpty(roundEnd))
             {
                 var roundEndLabel = new RichTextLabel();
                 roundEndLabel.SetMarkup(Loc.GetString(roundEnd));
-                RoundEndSummaryTab.AddChild(roundEndLabel);
+                roundEndSummaryContainer.AddChild(roundEndLabel);
             }
 
             //Duration
             var roundTimeLabel = new RichTextLabel();
             roundTimeLabel.SetMarkup(Loc.GetString("round-end-summary-window-duration-label",
-                                                   ("hours",roundTimeSpan.Hours),
-                                                   ("minutes",roundTimeSpan.Minutes),
-                                                   ("seconds",roundTimeSpan.Seconds)));
-            RoundEndSummaryTab.AddChild(roundTimeLabel);
+                                                   ("hours", roundDuration.Hours),
+                                                   ("minutes", roundDuration.Minutes),
+                                                   ("seconds", roundDuration.Seconds)));
+            roundEndSummaryContainer.AddChild(roundTimeLabel);
```

Comment:
It was already like this, but IMO the round time information should probably appear before the list of traitors. Currently it sort of makes it look like that's how long the traitor survived.

i.e. maybe move:
![move_duration](https://user-images.githubusercontent.com/60421075/145703279-07abf659-0112-478e-82ea-7fe6570a944c.png)

Reply:
I moved the text, and removed a Loc.GetString that would always fail, because it gets a big blob of text.